includes/deprecated-functions.php

```diff
@@ -1157,3 +1157,27 @@ function edd_record_status_change( $payment_id, $new_status, $old_status ) {
 
 	edd_insert_payment_note( $payment_id, $status_change );
 }
+
+/**
+ * Legacy pre-refund hook which fired after a payment status changed, but before store stats were updated.
+ *
```

Comment:
Can you add to both docblocks something like:

```php
 * @todo Formally deprecate in EDD 3.1
```

Reply:
Maybe also include a `@link` to this issue: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/8574